app/controllers/users_controller.rb

```diff
@@ -14,18 +22,52 @@ def create
       flash[:success] = "Welcome to the Sample App!"
       redirect_to @user
     else
-      flash[:danger] = "Danger! Please try again."
+      flash[:danger] = "Sorry. Please try again."
       render :new
     end
   end
 
   def edit
-    @user = User.find params[:id]
+    @user = User.find_by params[:id]
+  end
+
+  def update
+    @user = User.find_by params[:id]
```

Comment:
define a method load_user to find user. and use before_action to call it
EX:
before_action :load_user, only: [:edit, :update, :destroy]
def load_user
@user = User.find_by params[:id]
end